tests/testthat/test_base_multilabel.R

```diff
@@ -61,6 +61,18 @@ test_that("MultilabelBinaryRelevanceWrapper with glmnet", {
   expect_error(pred, NA)
 })
 
+test_that("check unique id", {
+  lrn = makeLearner("classif.rpart")
+  id1 = getLearnerId(makeMultilabelBinaryRelevanceWrapper(lrn))
```

Comment:
you could also directly add a test to check if the id contaisn the expected string, e.g. `expect_string(id1, pattern = "multilabel.br.classif.rpart")`